docs/current_docs/api/remote-modules.mdx

```diff
@@ -37,7 +37,36 @@ dagger -m git@github.com:kpenfound/dagger-modules/golang@v0.2.0 call \
   export --path=./build
 ```
 
-## Configuring SSH authentication
+## Authentication Methods
+
+Dagger supports both HTTPS and SSH authentication for accessing remote repositories.
+
+### HTTPS Authentication
+
+For HTTPS authentication, Dagger uses your system's configured Git credential manager. This means if you're already authenticated with your Git provider, Dagger will automatically use these credentials when needed.
+
+The following credential helpers are supported:
+- Git Credential Manager
+- macOS Keychain
+- Windows Credential Manager
+- Custom credential helpers configured in your `.gitconfig`
+
+To verify your credentials are properly configured, try cloning a private repository:
+
+```shell
+git clone https://github.com/username/private-repo.git
```

Comment:
nit: might be worth clarifying this isn't a real repo and should be replaced with your own